chemie/corporate/templates/corporate/interview_detail.html

```diff
@@ -3,73 +3,80 @@
+            <span class="badge badge-pill
             {% if specialization.id == 1 %} badge-custom-1 {% endif %}
             {% if specialization.id == 2 %} badge-custom-2 {% endif %}
             {% if specialization.id == 3 %} badge-custom-3 {% endif %}
             {% if specialization.id == 4 %} badge-custom-4 {% endif %}
             {% if specialization.id == 5 %} badge-custom-5 {% endif %}
             {% if specialization.id == 6 %} badge-custom-6 {% endif %}
              ">{{ specialization }}</span>
-        {% endfor %}
+          {% endfor %}
+        </div>
       </div>
     </div>
   </div>
-</div>
 
-<div class="container">
-  <div class="row" style="margin-bottom: 100px">
-    <div class="col-12 col-lg-1 order-3 order-lg-1"></div>
-    <div class="col-12 col-lg-6 order-2 order-lg-2">
-      {{ interview.text | safe }}
-    </div>
-    <div class="col-12 col-lg-4 order-1 order-lg-3">
-      <img class="img-fluid" style="margin-bottom: 30px; width: 100%; max-width: 60vw"
-           src="{{ interview.picture.url }}">
-    </div>
-    <div class="col-12 col-lg-12 order-4">
-      {% if perms.corporate.delete_interview %}
-      <a type="button" class="button-warning"
-         onclick="deleteInterviewPopup({{ interview.id }})"
-         id="deleteInterview{{ interview.id }}"
-         data-url="{% url "corporate:interview_delete" interview.id %}">
-      Slett intervju
-      </a>
-      {% endif %}
-      {% if perms.corporate.edit_interview %}
-      <a type="button" class="button-fill"
-         id="editInterview{{ interview.id }}"
-         href="{% url "corporate:interview_edit" interview.id %}">
-      Endre intervju
-      </a>
-      {% endif %}
+  <div class="container">
+    <div class="row" style="margin-bottom: 100px">
+      <div class="col-12 col-lg-1 order-3 order-lg-1"></div>
+      <div class="col-12 col-lg-6 order-2 order-lg-2">
+        {{ interview.text | safe }}
+        <br>
+        {% if perms.corporate.edit_interview %}
+          <a type="button" class="button-fill"
+             id="editInterview{{ interview.id }}"
+             href="{% url "corporate:interview_edit" interview.id %}">
+            Endre intervju
+          </a>
+        {% endif %}
+        {% if perms.corporate.delete_interview %}
+          <a type="button" class="button-warning"
+             onclick="deleteInterviewPopup({{ interview.id }})"
+             id="deleteInterview{{ interview.id }}"
+             style="float:right"
+             data-url="{% url "corporate:interview_delete" interview.id %}">
+            Slett intervju
+          </a>
+        {% endif %}
+      </div>
+      <div class="col-12 col-lg-4 order-1 order-lg-3">
+        <img class="img-fluid" style="margin-bottom: 30px; width: 100%; max-width: 30vw"
+             src="{{ interview.picture.url }}">
+        <img class="img-fluid" style="margin-bottom: 30px; width: 100%; max-width: 30vw"
+             src="{{ interview.company_picture.url }}">
```

Comment:
This causes an exception when there is no company_picture associated with the interview.  This will be a problem for existing interviews that are missing a company picture.

Reply:
Good catch! I have added a check now.